### PR TITLE
[Merge] load render option in atom render plugin and use render actor setting in atom debug draw

### PR DESCRIPTION
**Pull request by amzn-rhhong**
https://user-images.githubusercontent.com/69218254/140016230-58a88474-4d90-4ee3-af7f-e60dc84eaecf.mp4

### CODE REVIEW

File: Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp [hunk 1]
```diff
@@ -83,19 +85,27 @@ namespace AZ::Render
...
+                    RenderWireframe(mesh, globalTM, scaleMultiplier);
                 }
             }
         }
     }
 
+    float AtomActorDebugDraw::CalculateScaleMultiplier(EMotionFX::ActorInstance* instance) const
+    {
+        const AZ::Aabb aabb = instance->GetAabb();
+        const float aabbRadius = AZ::Vector3(aabb.GetMax() - aabb.GetMin()).GetLength() * 0.5f;
```
Comment (amzn-jillich) [inline Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp:104]: You can use aabb.GetExtents().GetLength() here

File: Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp [hunk 2]
```diff
@@ -218,7 +229,7 @@ namespace AZ::Render
         auxGeom->DrawLines(lineArgs);
     }
 
-    void AtomActorDebugDraw::RenderNormals(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM, bool vertexNormals, bool faceNormals)
+    void AtomActorDebugDraw::RenderNormals(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM, bool vertexNormals, bool faceNormals, float scaleMultiplier)
```
Comment (amzn-jillich) [inline Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp:232]: Can you add the color as parameter here and get access to the settings from the place who calls the RenderNormals() function please? Let's try to make this function a static and independent one.

File: Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp [hunk 3]
```diff
@@ -317,14 +325,14 @@ namespace AZ::Render
...
-            lineArgs.m_colors = &colorVertexNormals;
+            lineArgs.m_colors = &renderActorSettings.m_vertexNormalsColor;
             lineArgs.m_colorCount = 1;
             lineArgs.m_depthTest = RPI::AuxGeomDraw::DepthTest::Off;
             auxGeom->DrawLines(lineArgs);
         }
     }
 
-    void AtomActorDebugDraw::RenderTangents(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM)
+    void AtomActorDebugDraw::RenderTangents(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM, float scaleMultiplier)
```
Comment (amzn-jillich) [inline Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp:335]: Same here, color as parameter so the function can be static and independent.

File: Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp [hunk 4]
```diff
@@ -409,7 +413,7 @@ namespace AZ::Render
         auxGeom->DrawLines(lineArgs);
     }
 
-    void AtomActorDebugDraw::RenderWireframe(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM)
+    void AtomActorDebugDraw::RenderWireframe(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM, float scaleMultiplier)
```
Comment (amzn-jillich) [inline Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp:416]: Same here

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/RenderOptions.cpp [hunk 5]
```diff
@@ -318,6 +316,9 @@ namespace EMStudio
 
         options.m_manipulatorMode = static_cast<ManipulatorMode>(settings->value("manipulatorMode", options.m_manipulatorMode).toInt());
 
+        AZ::Render::RenderActorSettings& renderActorSettings = EMotionFX::GetRenderActorSettings();
+        options.CopyToRenderActorSettings(renderActorSettings);
```
Comment (amzn-jillich) [inline Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/RenderOptions.cpp:320]: Is this also being called when changing a color in the preferences window? Is that working? I couldn't see the on option changed callback being modified here.

**Review by amzn-jillich — APPROVED** [review]

**Review by AMZN-Gene — APPROVED** [review]

**Pull request merged**